templates/web/base/report/new/category_extras_fields.html

```diff
@@ -20,7 +20,9 @@
       <p class='form-error'>[% field_errors.$x_meta_name %]</p>
       [% END -%]
       [% IF meta.variable != 'false' %]
-        [% IF meta.datatype == 'multivaluelist' %]
+        [% IF meta.datatype == 'datetime' %]
```

Comment:
Will get to this after the mainline integration work if that's alright

Reply:
On second thoughts, I'm not sure about this, it seems like Shropshire and the fixture cats want just a `date` rather than a `datetime` - perhaps this should be its own open311 field type?